project/build.scala

```diff
@@ -45,18 +46,24 @@ object BananaRdfBuild extends Build {
     .settings(unidocSettings:_*)
 
   lazy val banana_jvm = bananaM
-    .project(Jvm, rdf_jvm, rdfTestSuite_jvm, ntriples_jvm, plantain_jvm, jena, sesame, examples)
+    .project(Jvm, rdf_jvm, rdfTestSuite_jvm, ntriples_jvm, plantain_jvm, jena, sesame)//, examples)
```

Comment:
I'm sorry to be such a hindrance.

I removed the example there, and commented out the project later, as half way to implementing the suggestion in https://github.com/w3c/banana-rdf/pull/185, to move the examples to their own build file. As this is not yet done, perhaps we could re-include the examples